Assets/Scripts/Model/FfbEnumerationFactory.cs

```diff
@@ -43,6 +43,9 @@ public static void RegisterModifier(Type type, string key, FfbEnumerationFactory
         public static T As<T>(this FFBEnumeration ffbEnum)
             where T : FfbEnumerationFactory
         {
+            if ((ffbEnum is null) || (ffbEnum.key is null)){
```

Comment:
A quick style comment - The following would look cleaner for this line:
 if (ffbEnum?.key is null) {